INSTALL.md

```diff
@@ -125,7 +125,7 @@ In case none of the binary packages listed on our website work for your system,
 The standard GNU/Linux tools and Python are minimal requirements to compile Shogun.
 To compile the interfaces, in addition to [swig](http://www.swig.org/) itself, you will need language specific development packages installed, see [interfaces](#manual-interfaces) below.
 
-There is a larger number of optional requirements.
+There is a larger number of optional requirements.Install dependencies like [ply](https://pypi.python.org/pypi/ply) ,[ctags](https://packagecontrol.io/packages/CTags)  if not found in the output lists of cmake. 
```

Comment:
It also makes more sense to me to have this new sentence after the one below "The output of cmake output lists optional ...". By the way, in that sentence we can also get rid of the second 'output' since it is redundant.